web/src/js/ducks/flows.js

```diff
@@ -210,5 +210,8 @@ export function updateFlow(item) {
  * @private
  */
 export function removeFlow(id) {
-    return { type: REMOVE, id }
+    return (dispatch) => {
+        dispatch(select())
```

Comment:
I'd prefer if removing a flow would select the next one, so that I can remove a bunch of flows by pressing `d` repeatedly.